services/ui-src/src/types/reportContext.ts

```diff
@@ -9,21 +9,29 @@ export interface ReportKeys {
 }
 
 export interface ReportMetadataShape extends ReportKeys {
-  submissionCount?: number;
-  reportType: string;
+  // Main Report Information
   submissionName: string;
   status: ReportStatus;
+  // Who Touched It/Submitted It
   createdAt: number;
   lastAltered: number;
   lastAlteredBy: string;
   submittedBy?: string;
   submitterEmail?: string;
   submittedOnDate?: number;
-  archived?: boolean;
-  locked?: boolean;
+  // Time Information
   dueDate: number;
   reportPeriod: number;
-  finalSar?: boolean;
+  reportYear: number;
+  // Admin Use Cases
+  archived?: boolean;
+  locked?: boolean;
+  submissionCount?: number;
+  // Connection to let Sar forms know what Work Plan they use to copy data from and vice versa
+  sar?: string;
```

Comment:
[very small nit, non-blocking]: i feel like `sar` and `workPlan` might get misinterpreted out of context.  I wonder if it would be more clear if we called these something like `associatedSar` or `associatedWorkPlan`?